components/x-teaser/src/concerns/image-service.js

```diff
@@ -1,14 +1,19 @@
 const BASE_URL = 'https://www.ft.com/__origami/service/image/v2/images/raw';
-const OPTIONS = ['source=next', 'fit=scale-down', 'dpr=2'];
+const OPTIONS = Object.freeze({
+	source: 'next', 
+	fit: 'scale-down',
+	dpr:2,
+});
 
 /**
  * Image Service
  * @param {String} url
  * @param {Number} width
  * @param {String} options
  */
-export default function imageService(url, width, options) {
+export default function imageService(url, width, addOptions, overrideOptions={}) {
```

Comment:
Maybe not one for now, but it seems that `addOptions` and `overrideOptions` are almost the same. I wonder if we should replace `addOptions` with `overrideOptions` as the way you've added `overrideOptions` is more flexible and handles both scenarios. What do you think?

It looks like we're only using the previous options argument in one other place (Headshot component) to add a tint which could also be done with `overrideOptions`.

Reply:
thanks for checking that, I wasn't 100% confident that it was the only dependency, yeah it'd be nice be rid of addOptions altogether. Not entirely out of the scope if we can be sure nothing else will break!